### PR TITLE
Add protobuf parse for live tracking

### PR DESCRIPTION
**Pull request by alanna-zhou**
## Overview
TCAT recommended that developers use the protobuf API to get live tracking data, because the JSON debug version is unstable!

## Changes Made
I used the protobuf compiler to get the python file that defines the structure of the protobuf message structure of TCAT's live tracking (gtfs-realtime) info. This was the python protobuf API that I was able to use to parse the live tracking data!

However, the auto-compiled file (gtfs_realtime_pb2.py) doesn't pass flake?! What to do about this plz give me advice @meganle @aliciaxw 

## Test Coverage
Works locally and will be letting this sit on the dev server.

## Next Steps 
Will be working on converting our alerts info to be using protobuf as well.

## Related PRs or Issues
#47

### CODE REVIEW

**Comment by meganle** [comment]
You can probably configure flake to ignore that file. That’s prob the best option ?

File: gtfs_realtime_pb2.py [hunk 1]
```diff
@@ -0,0 +1,1870 @@
...
... \x01(\t\x12\x14\n\x0c\x64irection_id\x18\x06 \x01(\r\x12\x12\n\nstart_time\x18\x02 \x01(\t\x12\x12\n\nstart_date\x18\x03 \x01(\t\x12T\n\x15schedule_relationship\x18\x04 \x01(\x0e\x32\x35.transit_realtime.TripDescriptor.ScheduleRelationship"O\n\x14ScheduleRelationship\x12\r\n\tSCHEDULED\x10\x00\x12\t\n\x05\x41\x44\x44\x45\x44\x10\x01\x12\x0f\n\x0bUNSCHEDULED\x10\x02\x12\x0c\n\x08\x43\x41NCELED\x10\x03*\x06\x08\xe8\x07\x10\xd0\x0f"M\n\x11VehicleDescriptor\x12\n\n\x02id\x18\x01 \x01(\t\x12\r\n\x05label\x18\x02 \x01(\t\x12\x15\n\rlicense_plate\x18\x03 \x01(\t*\x06\x08\xe8\x07\x10\xd0\x0f"\x92\x01\n\x0e\x45ntitySelector\x12\x11\n\tagency_id\x18\x01 \x01(\t\x12\x10\n\x08route_id\x18\x02 \x01(\t\x12\x12\n\nroute_type\x18\x03 \x01(\x05\x12.\n\x04trip\x18\x04 \x01(\x0b\x32 .transit_realtime.TripDescriptor\x12\x0f\n\x07stop_id\x18\x05 \x01(\t*\x06\x08\xe8\x07\x10\xd0\x0f"\x96\x01\n\x10TranslatedString\x12\x43\n\x0btranslation\x18\x01 \x03(\x0b\x32..transit_realtime.TranslatedString.Translation\x1a\x35\n\x0bTranslation\x12\x0c\n\x04text\x18\x01 \x02(\t\x12\x10\n\x08language\x18\x02 \x01(\t*\x06\x08\xe8\x07\x10\xd0\x0f*\x06\x08\xe8\x07\x10\xd0\x0f\x42\x1d\n\x1b\x63om.google.transit.realtime',
```
[the remark this answers is not in the document]
Comment (alanna-zhou) [reply gtfs_realtime_pb2.py]: breh this is auto gen'd code what should i do abt this

**Comment by alanna-zhou** [comment]
> You can probably configure flake to ignore that file. That’s prob the best option ?

ohhh okie

File: gtfs_realtime_pb2.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (young-k) [reply gtfs_realtime_pb2.py]: hm, can't you ignore specific files in black or through hound?

**Pull request merged**